lib/nostrum/struct/message.ex

```diff
@@ -107,6 +109,20 @@ defmodule Nostrum.Struct.Message do
   """
   @type member :: Member.t() | nil
 
+  @typedoc """
+  reference data sent with crossposted messages and replies
```

Comment:
For consistency could we capitalize the first words of these sentences?

Reply:
Ah, sure.  I just pasted these from the discord docs.